catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -656,6 +706,48 @@ public void testListFilesets() throws IOException {
     Assertions.assertEquals(fileset2.name(), nameIdentifiers1[1].name());
   }
 
+  @Test
+  public void testListFilesetFiles() throws IOException {
+    // clear schema
+    dropSchema();
+    createSchema();
+
+    String filesetName = "test_list_fileset_files";
+    String storageLocation = storageLocation(filesetName);
+    createFileset(filesetName, "comment", MANAGED, storageLocation, ImmutableMap.of("k1", "v1"));
+    assertFilesetExists(filesetName);
+
+    Path basePath = new Path(storageLocation);
+    fileSystem.mkdirs(basePath);
+
+    String fileName = "test1.txt";
+    Path file1 = new Path(basePath, fileName);
+    try (FSDataOutputStream out = fileSystem.create(file1, true)) {
+      out.write("hello".getBytes(StandardCharsets.UTF_8));
+    }
+
+    NameIdentifier filesetIdent = NameIdentifier.of(schemaName, filesetName);
+    String actualJson = getFileInfos(filesetIdent, null, null);
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
There is no need for deserialization; you can directly asset the equals of string values.